src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.cpp

```diff
@@ -99,7 +99,9 @@ void MultiDiracDeterminant::evaluateForWalkerMove(const ParticleSet& P, bool fro
 {
   evalWTimer.start();
   if (fromScratch)
+  {
```

Comment:
{} not needed. but please leave an empty line after.

Reply:
fixed